libbeat/generator/fields/fields.go

```diff
@@ -53,6 +48,12 @@ func collectCommonFiles(esBeatsPath, beatPath string, fieldFiles []*YmlFile) ([]
 		}
```

Comment:
Can we also do `ecs.yml` first on line 41 so common in Beats could overload it?

Reply:
Sure, though I wouldn't expect that to happen - we'd always move something from common to ecs right?

Reply:
Yes, but common could have a more specific description for something then ECS.

Reply:
Good point!  Updating